### ISSUE TITLE
Problem saving roundcube email & password

### ISSUE THREAD

**Issue by Grunthos** [hypery2k/owncloud#362]
When i click 'Update Email Identity' the progress message is displayed: 'Saving...', and it never goes away.

Looking in the logs I see:

```
Error 	roundcube 	RoundcubeLogin.class.php: openUrlConnection: \n Opening url https:///roundcube/ failed with Could not resolve host: https 
Error 	roundcube 	RoundcubeLogin.class.php: openUrlConnection: \n Opening url https:///roundcube/ failed with Could not resolve host: https 
Fatal 	index 	
Exception: {"Exception":"OC_Mail_NetworkingException","Message":
	"Unable to determine network-status due to technical problems.","Code":0,"Trace":"
#0 \/srv\/www\/htdocs\/owncloud\/apps\/roundcube\/lib\/RoundCubeLogin.class.php(279): 
	OC_RoundCube_Login->sendRequest('\/roundcube\/', Array)\n
#1 \/srv\/www\/htdocs\/owncloud\/apps\/roundcube\/lib\/RoundCubeApp.class.php(357): 
	OC_RoundCube_Login->login('MYUSER', 'MYPASS...')\n
#2 \/srv\/www\/htdocs\/owncloud\/apps\/roundcube\/lib\/RoundCubeApp.class.php(530): 
	OC_RoundCube_App::login('', '', '\/roundcube\/', 'MYUSER', 'MYPASS...')\n
#3 \/srv\/www\/htdocs\/owncloud\/apps\/roundcube\/ajax\/userSettings.php(10): 
	OC_RoundCube_App::saveUserSettings('roundcube', 'MYUSER', 'MYUSER', 'MYPASS...')\n
#4 \/srv\/www\/htdocs\/owncloud\/lib\/private\/route\/route.php(154) : 
	runtime-created function(1): require_once('\/srv\/www\/htdocs...')\n
#5 [internal function]: 
	__lambda_func(Array)\n
#6 \/srv\/www\/htdocs\/owncloud\/lib\/private\/route\/router.php(283): 
	call_user_func('\\x00lambda_183', Array)\n
#7 \/srv\/www\/htdocs\/owncloud\/lib\/base.php(874): OC\\Route\\Router->match('\/apps\/roundcube...')\n
#8 \/srv\/www\/htdocs\/owncloud\/index.php(48): 
	OC::handleRequest()\n
#9 {main}","File":"\/srv\/www\/htdocs\/owncloud\/apps\/roundcube\/lib\/RoundCubeLogin.class.php","Line":434} 
```
I'm using OC 8.1.5 and a recently downloaded roundcube app. Roundcube is version 1.1.3. Roundcube works fine by itself, as does ownCloud.

They are at:

Roundcube:  /srv/www/roundcubemail
with an alias /roundcube pointing to this location

ownCloud: /srv/www/htdocs/owncloud

**Comment by Grunthos** [comment]
Temporarily upgraded to ownCloud 8.2.1 and experimental roundcube, but got same error. Reverted to oc 8.1.5.

FWIW, allow_url_fopen  is On for cli and apache2.

**Comment by Grunthos** [comment]
OK...this may be solved. Despite them being on the same host, I seem to need to manually set the roundcube hosts in the roundcube admin settings.

**Comment by hypery2k** [comment]
this is fixed in the master, see 19d83d1957d28f025c01fca0fb90cd352bb723d6
